src/containers/account/components/TransferList.js

```diff
@@ -88,7 +91,7 @@ class TransferList extends React.Component {
         }
 
         return (
-            <Segment loading={isLoading} style={{ color: "black" }}>
+            <Segment style={{ color: "black" }}>
```

Comment:
The `noItemMessage` is visible now while it's loading the initial set of history. And also  no indication for the user that anything is being loaded in that case.
I think because `loading={isLoading}` has been moved from here.
Try switch between accounts with and without balance or just reload the page. 
We should show a loading indicator for the initial fetch too and should not show noItemMessage while loading it.

Reply:
good catch, thanks! fixed :)